i18n/de.json

```
@@ -2116,10 +2389,24 @@
   "announcement_bar.error.past_grace": "Enterprise Lizenz ist abgelaufen und einige Funktionen wurden evtl. deaktiviert. Bitte wenden Sie sich an den Systemadministrator für mehr Details.",
   "announcement_bar.error.preview_mode": "Vorschaumodus: E-Mail-Benachrichtigungen wurden noch nicht konfiguriert.",
+  "announcement_bar.error.purchase_a_license_now": "Erwerben Sie jetzt eine Lizenz",
   "announcement_bar.error.site_url.full": "Bitte konfigurieren Sie ihre [Site-URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in der [Systemkonsole](/admin_console/environment/web_server).",
   "announcement_bar.error.site_url_gitlab.full": "Bitte konfigurieren Sie ihre [Site-URL](https://docs.mattermost.com/administration/config-settings.html#site-url) in der [Systemkonsole](/admin_console/environment/web_server) oder in gitlab.rb, falls Sie Gitlab-Mattermost verwenden.",
+  "announcement_bar.error.trial_license_expiring": "**Es sind {days} Tage verfügbar in Ihrem freien Test.**",
+  "announcement_bar.error.trial_license_expiring_last_day": "**Dies ist der letzte Tag Ihres freien Tests. Kaufen Sie jetzt eine Lizenz um die E10 & E20 Features weiter zu nutzen.**",
   "announcement_bar.error.warn_metric_status.link": "Mehr erfahren",
   "announcement_bar.notification.email_verified": "E-Mail-Adresse bestätigt",
+  "announcement_bar.number_active_users_warn_metric_status.text": "Sie haben jetzt über {limit} Benutzer. Wir empfehlen dringend die Nutzung von erweiterten Funktionen für große Server.",
+  "announcement_bar.number_of_posts_warn_metric_status.text": "Sie haben mehr als {limit} Beiträge. We raten dringend zur Nutzung von erweiterten Features um Leistungseinbußen zu vermeiden.",
+  "announcement_bar.warn.contact_support_text": "Um Ihre Lizenz zu erneuern, kontaktieren Sie den Support unter support@mattermost.com.",
+  "announcement_bar.warn.email_support": "[Support kontaktieren](!{email}).",
+  "announcement_bar.warn.no_internet_connection": "Es sieht so aus, als ob keine Verbindung zum Internet besteht.",
+  "announcement_bar.warn.renew_license_now": "Lizenz jetzt erneuern",
+  "announcement_bar.warn_metric_status.number_of_posts.text": "Sie haben jetzt mehr als 2.000.000 Beiträge. Wir raten dringend dazu, erweiterte Feature zu nutzen um Leistungseinbußen zu vermeiden.",
+  "announcement_bar.warn_metric_status.number_of_posts_ack.text": "Danke, dass Sie Mattermost kontaktiert haben. Wie setzen uns kurzfristig mit Ihnen in Verbindung.",
+  "announcement_bar.warn_metric_status.number_of_users.text": "Sie haben über 500 Benutzer. Wir raten dringend dazu, erweiterte Feature für große Server zu nutzen.",
+  "announcement_bar.warn_metric_status.number_of_users_ack.text": "Danke, dass Sie Mattermost kontaktiert haben. Wie setzen uns kurzfristig mit Ihnen in Verbindung.",
+  "announcement_bar.warn_metric_status_ack.text": "Danke, dass Sie Mattermost kontaktiert haben. Wie setzen uns kurzfristig mit Ihnen in Verbindung.",
```

```suggestion
  "announcement_bar.warn_metric_status_ack.text": "Danke, dass Sie Mattermost kontaktiert haben. Wir setzen uns kurzfristig mit Ihnen in Verbindung.",
```
Would change in Weblate first however and then just push the changes again.

---

Fixed in weblate :)